src/features/articles/components/article-editor/editor-config.ts

```diff
@@ -75,6 +75,7 @@ import {
   WordCount,
 } from "ckeditor5";
 import {FileExplorerPlugin} from "./plugins/file-explorer-plugin";
+import ExecutableCodeBlock from "@/features/articles/components/article-editor/plugins/executable-code-block";
```

Comment:
can we move it to `./plugins/executable-code-block` ?

I assume that we follow the same conventions in the whole code.